kotlin-native/backend.native/tests/objcexport/coroutines.swift

```diff
@@ -21,6 +21,33 @@ private func testCallSimple() throws {
     try assertNil(error)
 }
 
+private func testUnitCallSimple() throws {
+#if LEGACY_SUSPEND_UNIT_FUNCTION_EXPORT
+    var result: KotlinUnit? = nil
+#endif
+    var error: Error? = nil
+    var completionCalled = 0
+
+#if LEGACY_SUSPEND_UNIT_FUNCTION_EXPORT
+    CoroutinesKt.unitSuspendFun { _result, _error in
+        completionCalled += 1
+        result = _result
+        error = _error
+    }
+#else
+    CoroutinesKt.unitSuspendFun { _error in
+        completionCalled += 1
+        error = _error
+    }
+#endif
+
+#if LEGACY_SUSPEND_UNIT_FUNCTION_EXPORT
+    try assertSame(actual: result, expected: KotlinUnit.shared)
```

Comment:
Q: should I change `KotlinUnit()`throughout the file to this for consistency?

Reply:
Thanks! I don't think this is required. We will probably do this later, when (if?) we decide to keep only `KotlinUnit.shared`.